src/lib.rs

```diff
@@ -1,24 +1,26 @@
 #![allow(clippy::module_inception)]
 #![allow(clippy::mutable_key_type)]
 mod app_context;
-pub mod async_graphql_hyper;
-pub mod blueprint;
+mod async_graphql_hyper;
+mod blueprint;
+pub mod builder;
 pub mod cache;
 #[cfg(feature = "cli")]
 pub mod cli;
 pub mod config;
-pub mod data_loader;
+mod data_loader;
 pub mod directive;
```

Comment:
only the builder should be public IMO. Everything else we can make private.

Reply:
they are used in tests and benches

Reply:
Let's add it to our todo list so that we can do it later. We should only make the builder public ideally. Rest of the stuff should remain private to the library.